scripts/uploader.rb

```diff
@@ -52,11 +52,34 @@ def run
         )
 
         puts "Uploading #{appliance} as #{destination_name}...complete: #{destination_url}"
+
+        next unless master?(appliance)
+
+        if nightly?
+          devel = devel_filename(destination_name)
+          puts "Copying   #{appliance} to #{devel}..."
+
+          RestClient::Request.execute(
+            :method  => :copy,
+            :url     => destination_url,
+            :headers => token_headers.merge("Destination" => "/#{container}/#{devel}")
+          )
+
+          puts "Copying   #{appliance} to #{devel}...complete"
+        end
       end
     end
 
     private
 
+    def master?(filename)
+      filename.include?("-master-")
+    end
+
+    def nightly?
+      !release?
```

Comment:
We do support `type` other than `nightly` and `release` (although I have never personally used it....), so wouldn't it be better to explicitly check for `type == "nightly"`?

This script is currently called only for nightly/release, so the end result will be same, though...